api/jobs/models/job.go

```diff
@@ -86,12 +86,6 @@ type Job struct {
 	// required: false
 	PromotedFromDeployment string `json:"promotedFromDeployment,omitempty"`
 
-	// PromotedDeploymentName the name of the deployment that was promoted
-	//
-	// required: false
-	// example: component-6hznh
-	PromotedDeploymentName string `json:"promotedDeploymentName,omitempty"`
```

Comment:
This is used in radix web-console, but seems as a bug the property`PromotedFromDeployment` is populated instead in `radix-api` . Either `PromotedDeploymentName` should replace `PromotedFromDeployment` here or usage of `PromotedDeploymentName` should be replaced with `PromotedFromDeployment` in radix web-console

Reply:
Thanks, yes I noticed today and talked to Nils about it, the web-console is updated now to use `PromotedFromDeployment`